.gitignore

```diff
@@ -20,6 +20,7 @@ __pycache__/
 *.xlsx
 *.pbz2
 *.docx
+.env
```

Comment:
Are .env files being used here?

Environments should be used via anaconda / miniforge.

Reply:
I've always used .env files to store hardcoded sensitive info - API keys, DB info, Filepaths etc. I think its a bit less cumbersome than config files for portability. Will leave this call up to you. Can change if needed.

Reply:
move over to yaml